### PR TITLE
Feature/12 selector implementation

### PR DESCRIPTION
**Pull request by fjcalzado**

### CODE REVIEW

File: 00_ES6/05_redux_selector/README.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+For this sample, we are going to add a new functionality to our chat: message search. For that purpose, we must create a few things first.
+
+You can start from the previous sample: 04_redux_saga.
+
+# Steps
+
+- Let's create a new component to search in our chat log. You can add the following new files:
+
+_./src/pages/chat/components/chat-search.component.jsx_
+```diff
```
Comment (brauliodiez) [inline 00_ES6/05_redux_selector/README.md]: If it's a new component, instead of _diff_ use _javascript_

Comment (brauliodiez) [reply 00_ES6/05_redux_selector/README.md]: Please apply this to all code in readme, thumb rule:

- If it's new file, use always javascript, no diff (sintax highlight).
- If it's an update on a file (remove lines, add line), use diff

File: 00_ES6/05_redux_selector/README.md [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+
+_./src/pages/chat/chat.component.jsx_
+
+Import it:
+```diff
+  SendMessageActions,
+  ChatHeaderComponent,
+  ChatLogComponent,
+- ChatAutoscrollComponent
++ ChatAutoscrollComponent,
```
Comment (brauliodiez) [inline 00_ES6/05_redux_selector/README.md]: Why do we need an autoscroll component?

**Comment by brauliodiez** [comment]
Good stuff, some minor comments on readme.md plus a question on a scroll component.

File: 00_ES6/05_redux_selector/README.md (hunk 1 above)
Comment (fjcalzado) [reply 00_ES6/05_redux_selector/README.md]: Done. fixed in several places and committed.

File: 00_ES6/05_redux_selector/README.md (hunk 2 above)
Comment (brauliodiez) [reply 00_ES6/05_redux_selector/README.md]: cool, thanks for the clarification

**Pull request merged**